app/assets/javascripts/application.js

```diff
@@ -3,11 +3,11 @@
 
 //= require jquery
 //= require jquery_ujs
-//= require jquery-ui/datepicker
-//= require jquery-ui/draggable
-//= require jquery-ui/droppable
-//= require jquery-ui/tabs
-//= require jquery-ui/dialog
+//= require jquery-ui/widgets/datepicker
+//= require jquery-ui/widgets/draggable
```

Comment:
I'm not actually sure we use jquery's draggable/droppable. I think there are some references within timeline.js where there is some interface for rescheduling by drag/dropping reservations, but we don't actually do anything with that interaction.

I think it was a feature that I was experimenting with when we first built the timeline view, but we never used it.

We don't have to right now, but it wouldn't really be a bad idea to remove the dead code.